docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -44,28 +44,24 @@ This could be done a few different ways, but there are some explicit goals that
 - **Speed** - The time/number of steps required to build 2xx+ should be minimized.
 - **Distro partner/MSFT build unification** - We should minimize differences between the distro partner and MSFT builds.
 
-To achieve 2xx+ builds, we will use the "classic" dependency flow methodology that .NET has used for many years. The VMR's root `Version.Details.xml`/`Versions.props` files will be updated to include all dependencies used to "bootstrap" the .NET build. This includes 1xx branches. *Note: Despite having a reduced set of repos, 2xx+ branches will not have a "reduced" set of dependencies.* The intention here is that this is essentially a representation of the "previously source-built" concept that is present in source-build. We are just extending this concept to Microsoft's distribution, and making the representation explicit.
+To achieve 2xx+ builds, we will use the "classic" dependency flow methodology that .NET has used for many years but with a variation to account for VMR-to-VMR flow. When flow happens from the the VMR's 1xx to 2xx branch, all dependencies across all relevant repos will be updated in one PR. This ensures coherency. The dependencies being updated are applied to the individual repos' Version.Details.xml files. There will not be a root VMR Version.Details.xml that aggregates all of these dependencies.
 
-When building a repo within the VMR, the following version flow will be applied:
+Let's use roslyn's dependency on command-line-api as an example to demonstrate the flow.
```

Comment:
@premun What do you think a feature like this would cost in Maestro? Essentially, when 1xx flows to 2xx, we would have subscription metadata that describes the flow to each of the 2xx VMR repo dirs, complete with ExcludeAssets.

Reply:
The main problem with putting the info in the root V.D.xml file is that it changes the build behavior for 2xx

Reply:
@mmitche I was sort of accounting for something like this 2 years ago when adding the `SourceDirectory` and `TargetDirectory` attributes on subscriptions.

Maybe we could allow non-code-enabled subscriptions to define a `TargetDirectory` and it would just hit that `src/[TargetDirectory]`?

I'd say 2-3 weeks of work?

Reply:
We'd also want these to come in as a single PR. We could batch them? But it seems possible to screw up.

Reply:
BTW, this work can be tracked with https://github.com/dotnet/dotnet/issues/1108